pallets/elections/src/lib.rs

```diff
@@ -112,10 +144,29 @@ pub mod pallet {
     #[pallet::call]
     impl<T: Config> Pallet<T> {
         #[pallet::weight((T::BlockWeights::get().max_block, DispatchClass::Operational))]
-        pub fn change_members(origin: OriginFor<T>, members: Vec<T::AccountId>) -> DispatchResult {
+        pub fn change_members(
```

Comment:
I would remove all the calls except this one. There are potential storage writes that could be optimized but it's not worth it in my opinion as we would get one call instead of four and we call it very infrequently so the storage write overhead is negligible.

Reply:
ok, done